.github/workflows/maven.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Deploy P2 Updatesite
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
+    - name: Replace Updatesite
+      run: |
+        rm -r updatesite
+        git config --local user.name "GitHub Action"
```

Comment:
While trying the find a solution why my jq-approach wasn't working, I found a couple of Github actions that by default specify "Github Action" with a no-reply email address for the commits.

For instance: https://github.com/marketplace/actions/configure-git-credentials

I think this is actually a good idea because you can see very easily that the commit is considered to be created by a Github action, not a human being user.

What do you think?

Reply:
<img width="338" alt="Screenshot 2020-06-17 at 22 57 55" src="https://user-images.githubusercontent.com/2838592/84949783-091fb780-b0ee-11ea-8252-4ba2dffce061.png">

This is what it looks like.

Reply:
Yes I’m +1 for that for sure (if it’s easy enough to do) :)

Reply:
Looks awesome :D